CMSgov/mint-app
# [EASI-3155] Support Links As Docs

# EASI-3155

## Changes and Description

- Update `document.spec.js` e2e test
- Changed verbiage in `src/i18n/en-US/draftModelPlan/documents.ts`
- Added new queries to incorporate new links for documents 
- added new `LinkingDocumentFormTypes` type in `src/types/files.ts`



## How to test this change

1. Navigate to add a new document
2. Ensure there is a new segmented button to `Link to document`
3. Fill out form 
4. Ensure it is working as expected and matches Figma
5. Check the table to see if new link shows `Visit` instead of `View`
6. Work through the form and also in solution details to ensure things are not broken 



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
